### ISSUE TITLE
Blog section links must be distinguishable without relying on colour

### ISSUE THREAD

**Issue by oraNod** [ansible-community/community-website#261]
Applies to linked text in the blog section

To solve this problem, you need to fix at least (1) of the following:

- The link has insufficient color contrast of 2.04:1 with the surrounding text. (Minimum contrast is 3:1, link text: #004080, surrounding text: #000000)
- The link has no styling (such as underline) to distinguish it from the surrounding text

**Comment by oraNod** [comment]
Here is the problem

https://webaim.org/resources/linkcontrastchecker/?fcolor=5CBEC1&bcolor=004080

![image](https://github.com/ansible-community/community-website/assets/32749632/17e562f6-3f3f-4de4-9acd-1f160f805cf6)

**Comment by oraNod** [comment]
Changing the blog anchor colour to `#3333FF;` fixes it

![image](https://github.com/ansible-community/community-website/assets/32749632/1585b967-0a13-41a6-81be-08bba637f063)

**Comment by oraNod** [comment]
@tiyiprh Hi, could you please take a look and suggest a colour for the anchor text in the Blogs section? https://ansible-community-website--266.org.readthedocs.build/en/266/

thanks!

**Comment by tiyiprh** [comment]
Hi @oraNod after talking with the team, to help with this problem across the whole page and keep consistency, we felt to just make all text links blue (#0066CC) and on hover they should turn the Ansible navy (#004080) and be underlined. For areas with an arrow, such as the "back to top" and "view more," the arrow should also be the blue (#0066CC) but should not be part of the link and would not have a hover state. Let me know what you think!

Updated sections:
<img width="946" alt="Screenshot 2023-10-06 at 10 14 00 AM" src="https://github.com/ansible-community/community-website/assets/98424339/99896e6c-ed7c-43b5-b49a-a1ca2cd01060">
<img width="948" alt="Screenshot 2023-10-06 at 10 14 23 AM" src="https://github.com/ansible-community/community-website/assets/98424339/f81be52e-5aa6-41fa-bd05-d69384da6a2e">
<img width="948" alt="Screenshot 2023-10-06 at 10 14 37 AM" src="https://github.com/ansible-community/community-website/assets/98424339/c2030f7b-c98d-45a2-b585-804d544ca2a0">

**Comment by oraNod** [comment]
@tiyiprh As mentioned in the PR I don't think navy passes the link contrast check: https://webaim.org/resources/linkcontrastchecker/?fcolor=000000&bcolor=FFFFFF&lcolor=102441

How about we just leave it the one colour `#0066CC` and use underline to highlight the fact it is a clickable link?

I've also moved the arrows out of the anchors so they should not be underlined or clickable.

**Comment by oraNod** [comment]
@tiyiprh Also I didn't change the colour of the tab menu links and left them as cyan to match the icons. Please let me know what you think. Cheers.

**Comment by tiyiprh** [comment]
> @tiyiprh As mentioned in the PR I don't think navy passes the link contrast check: https://webaim.org/resources/linkcontrastchecker/?fcolor=000000&bcolor=FFFFFF&lcolor=102441
> 
> How about we just leave it the one colour `#0066CC` and use underline to highlight the fact it is a clickable link?
> 
> I've also moved the arrows out of the anchors so they should not be underlined or clickable.

@oraNod That sounds good to us!

**Comment by tiyiprh** [comment]
> @tiyiprh Also I didn't change the colour of the tab menu links and left them as cyan to match the icons. Please let me know what you think. Cheers.

That's fine, thanks!
